src/Aquarius.ONE.ClientSDK/Common/Logbook/LogbookApi.cs

```diff
@@ -203,6 +226,30 @@ public async Task<List<ConfigurationNote>> GetLogbookEntriesAsync(string logbook
         public async Task<bool> UpdateLogbookEntryAsync(string logbookId, string entryId, string entry, 
             string externalSourceId, string externalUsername, DateTime entryTime, float? geoPointX = null,
             float? geoPointY = null, RecordAuditInfo recordAuditInfo = null, params string[] tags)
+        {
+            var result = await UpdateLogbookEntry2Async(logbookId, entryId, entry, externalSourceId,
+                externalUsername, entryTime, geoPointX, geoPointY, recordAuditInfo, tags);
+
+            return result != null;
+        }
+
+        /// <summary>
+        /// Edit an entry in a logbook
+        /// </summary>
+        /// <param name="logbookId">Identifier of the logbook in which to update the entry</param>
+        /// <param name="entryId">Identifier of the entry to be edited</param>
+        /// <param name="entry">Text of the entry to be edited, this text replaces any existing entry text</param>
+        /// /// <param name="externalSourceId">Value used to associate the note with external data</param>
+        /// <param name="externalUsername">Value used to associate the note with an external user</param>
+        /// <param name="entryTime">Timestamp to be associated to the entry, should be in UTC</param>
+        /// <param name="geoPointX">Optional - the WGS84 longitude of the location of this entry</param>
+        /// <param name="geoPointY">Optional - the WGS84 latitude of the location of this entry</param>
+        /// <param name="recordAuditInfo">Audit information to save</param>
+        /// <param name="tags">Any tags that should be associated to the entry, no spaces allowed, this list replaces any existing tags</param>
+        /// <returns>Boolean value indicating whether the logbookEntry was successfully updated</returns>
+        public async Task<ConfigurationNote> UpdateLogbookEntry2Async(string logbookId, string entryId, string entry,
```

Comment:
Rather than just adding a 2 on these methods would a suffix like `withAuditInfo` make sense?

Reply:
Ah, I guess I was not paying close enough attention.  I'm fine with 2 in  that case since the consumer can see the return type in intellisense.